tuna/miopen_tables.py

```diff
@@ -518,6 +518,41 @@ class BNGolden(BASE, GoldenMixin):
   kernel_group = Column(Integer, nullable=True)
 
 
+class BenchmarkTable(BASE, GoldenMixin):
+  """benchmark table for framework and model parameters"""
+  __tablename__ = "benchmark_table"
+  __table_args__ = (UniqueConstraint("framework",
+                                     "model",
+                                     "batchsize",
+                                     "gpu_number",
+                                     "driver_cmd",
+                                     name="uq_idx"),)
+
+  framework = Column(String(length=128), nullable=False)
+  model = Column(String(length=128), nullable=False)
+  batchsize = Column(Integer, nullable=False, server_default="32")
+  gpu_number = Column(Integer, nullable=True, server_default="1")  #nullable?
+  driver_cmd = Column(String(length=128), nullable=False)
+
+
+class BenchmarkPerfTable(BASE, GoldenMixin):
+  """benchmark table for performance parameters"""
+  __tablename__ = "benchmark_perf_table"
+  __table_args__ = (UniqueConstraint("kernel_time",
+                                     "solver",
+                                     "workspace_sz",
+                                     "rocm_v",
+                                     "miopen_v",
+                                     name="uq_idx"),)
+
+  config = Column(Integer, ForeignKey("benchmark_table.id"), nullable=False)
+  kernel_time = Column(DOUBLE, nullable=False, server_default="-1")
+  solver = Column(String(length=128), nullable=True, server_default="")
+  workspace_sz = Column(BigInteger, nullable=False)
+  rocm_v = Column(Integer, nullable=False)
+  miopen_v = Column(Integer, nullable=False)
+
```

Comment:
You benchmark_perf_table is also missing an FKey to the benchmark table. Something like this:
benchmark = Column(Integer, ForeignKey("benchmark_table.id"), nullable=False)